### PR TITLE
Improve error handing for batch post failing

### PR DESCRIPTION
**Pull request by apackin**
#### What's this PR do?
Improve error handing for batch post failing in order to assist in debugging " Sorry! WhenIWork is loading story.."
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-199
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-204

### CODE REVIEW

File: jobs/scheduling/RecurShifts.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ function recurNewlyCreatedShifts(optionalNoShiftsForTesting) {
         After batch of shifts is created, we want to publish all shifts. (Note that passing in the `published` param
         in the requests that are batched doesn't actually publish them; we need to make a separate request to another route.)
       **/
+      if (response.message && /error/.test(response.message)) console.log("Error in RecurShift Batch Response: ", response);
```
Comment (jrauschenberg) [inline jobs/scheduling/RecurShifts.js]: do we want a CONSOLE_WITH_TIME here?

**Comment by jrauschenberg** [comment]
Other than one minor comment, good to go. 👍

**Comment by apackin** [comment]
Thanks you, John! @tongxiang

**Comment by tongxiang** [comment]
Since this is an unusual way of catching errors, could you add a comment--afterward, good to merge. 👍

**Pull request merged**